vignettes/geocoding_simple_output.Rmd

```diff
@@ -0,0 +1,350 @@
+
+Next, we will look at patterns in city, state, and zipcode.
+```{r}
+# Check the frequency of city names
+table(sort(ga_5k$city))
+```
+```{r}
+# Check the frequency of zipcodes
+head(table(sort(ga_5k$zipcode)), 100)
+```
+
+```{r}
+length(which(is.na(ga_5k$zipcode) == TRUE))
+```
+11 voters are missing zipcodes.
+
+```{r}
+length(which(is.na(ga_5k$zipcode) == TRUE & ga_5k$street_name == "MISSING ADDRESS"))
+```
+6 voters are missing zipcodes and street names.
+
+```{r}
+zip_miss <- which(is.na(ga_5k$zipcode) == TRUE & ga_5k$street_name == "MISSING ADDRESS")
+ga_5k <- ga_5k[-zip_miss, ]
+nrow(ga_5k)
+```
+There are now 4,989 registered voters in the voter file.
+
+There are also 9-digit zipcodes without hyphens in the voter file. Having zipcodes as hyphens may help the goeocoder be more accurate when trying to find an individual's corrdinates.
+```{r}
+source("~/github/geocodeCompare/geocoder_format.R")
+# Format zipcodes that are 9 digits long and treated as character variables
+ga_5k <- zip_hyphen(
+  voter_file = ga_5k,
+  voter_id = "registration_number",
+  zipcode = "zipcode"
+)
+```
+
+
+```{r}
+# Check that zzipcodes have been properly formatted
+head(table(ga_5k$zipcode), 6)
+```
+
+### Step 3: Prepare/Structure your voter data for geocoding.
+
+*Concatenate columns for street address.*
+```{r}
+ga_5k <- concat_streetname(
+  voter_file = ga_5k,
+  voter_id = "registration_number",
+  street_number = "street_number",
+  street_suffix = "street_suffix"
+)
+
+head(ga_5k$street_address, 6)
+```
+
+*Create a column for the final address in the voter file.*
+```{r}
+# Create a column for state. The GA voter registration dataset does not have a state column.
+ga_5k$state <- "GA"
+
+ga_5k <- concat_final_address(
+  voter_file = ga_5k,
+  voter_id = "registration_number",
+  street_address = "street_address",
+  city = "city",
+  state = "state",
+  zipcode = "zipcode"
+)
+```
+
+```{r}
+# convert dataframe into a tibble
+ga_5k <- as_tibble(ga_5k)
+
+head(ga_5k, 6)
+```
+
+
+### Step 4: run_geocoder() Function 
+### Select a geocoder and run the geocoder on the addresses in your file.
+
+*Select the geocoder you are going to use to find the geographies like coordinates (i.e. latitude and longitude) and FIPS codes for the addresses in the voter file.* 
+
+#### There are several options for geocoding your data using a geocoding API.  The eiCompare package utilizes the US Census Geocding API via a R package called censusxy.
+
+
+*Note: If you have more than 10000 voters in yoru file, we recommedn using parallel processing. More information on parallel processing can be found on the Parallel Processing vignette.*
+
+#### Let's start geocoding our data*
+*We recommend first geocoding your data with the US Census Geocoder API via the R package, censusxy.*
+
+voter file: A data frame contain the voter addresses, separated into columns for street, city, state, and zipcode
```

Comment:
Is this from the docstring? If so, it probably doesn't need to be here since the user can just look it up.